app/views/layouts/blacklight/base.html.erb

```diff
@@ -46,7 +46,11 @@
     <div class="ucb_container">
         <%= render partial: 'shared/header_navbar' %>
         <main id="main-container" class="<%= container_classes %>" role="main" aria-label="<%= t('blacklight.main.aria.main_container') %>">
-            <%= content_for(:container_header) %>
+            <span class="constraints-label sr-only visually-hidden"><%= t('blacklight.search.filters.title') %></span>
```

Comment:
This line shouldn't be here; it causes "You searched for:" to appear on every page, even pages that aren't search-related.

```suggestion
```